chain/chain/src/chain.rs

```diff
@@ -2424,56 +2426,69 @@ impl Chain {
         shard_tracker: &ShardTracker,
         me: &Option<AccountId>,
         parent_hash: &CryptoHash,
```

Comment:
`<unrelated rant>` Why do we have two names - prev and parent - for the same thing? 😢  `</unrelated rant> `